src/components/Button/editbutton.js

```diff
@@ -1,13 +1,13 @@
 import useStore from '../../hooks/useStore.js';
 
-import StyledEdit from './stylededit';
+import StyledIcon from './styledicon';
 
 export default function EditButton() {
 	const setEditing = useStore(state => state.setEditing);
 
 	return (
-		<StyledEdit onClick={() => setEditing(true)}>
+		<StyledIcon onClick={() => setEditing(true)}>
```

Comment:
You could do the same with this component 🙂

Reply:
Yes, I will do it here as well.